docs/specs/meta-clients/web3modal/api.md

```diff
@@ -1,142 +1,248 @@
+- `sa=`: SDK adapter
+  - Options: `evm-wagmi | evm-ethers | evm-ethers5`
+  - Example: `sa=evm-wagmi`
+- `sv=`: SDK version
+  - Options: `string`
+  - Example: `sv=5.0.0`
+- `projectId=`: The project ID for tracking and analytics
+  - Options: `string`
+  - Example: `projectId=702e2d45...`
 
 ## Headers
 
-All api requests must contain following headers. See [types](./types.md)
+All API endpoints expect the following simple (i.e. non-preflight-dependent) headers:
 
-```ts
-interface Headers {
-  "x-project-id": string;
-  "x-sdk-type": SdkType;
-  "x-sdk-version": SdkVersion;
-}
-```
+- `Accept-Encoding`
+  - Options: `gzip | br`
+  - Example: `Accept-Encoding: gzip, br`
 
-## Methods
+## API Endpoints
 
-### GET `/getWallets`
+### `POST /v2/wallets`
 
-Returns total count and wallet data, based on provided body inputs. See [types](./types.md).
+Fetches wallet data based on following request body provided as a stringified JSON object (`Content-Type: text/plain`).
 
-```ts
-interface Request {
-  headers: Headers;
-}
+#### Request Body
 
-interface UrlQueryParams {
-  page: string; // eg. 1
-  entries: string; // eg. 100
-  search?: string; // eg. MetaMa...
-  include?: string; // eg. id1,id2,id3
-  exclude?: string; // eg. id1,id2,id3
-  chains?: string; // eg. eip155:1,solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp
-  platform?: "ios" | "android";
+```typescript
+{
+  "page": number, // Required - The page number of results (e.g., `1`)
+  "entries": number, // Required - Number of entries per page (e.g., `100`)
+  "search": string | null, // Optional search term (e.g., `MetaMask`)
+  "include": string | null, // Optional comma-separated list of wallet IDs to include
+  "exclude": string | null, // Optional comma-separated list of wallet IDs to exclude
+  "chains": string | null, // Optional comma-separated list of chain identifiers (e.g., `eip155:1`)
+  "platform": string | null // Optional platform filter (`ios` or `android`)
 }
+```
+
+#### Response
 
-interface Response {
-  count: number;
-  data: Array<{
-    id: string;
-    name: string;
-    homepage: string;
-    image_id: string;
-    order: number;
-    mobile_link: string | null;
-    desktop_link: string | null;
-    webapp_link: string | null;
-    app_store: string | null;
-    play_store: string | null;
-    injected:
-      | {
-          namespace: string;
-          injected_id: string;
-        }[]
-      | null;
-  }>;
+```typescript
+{
+  "count": number,
+  "data": [
+    {
+      "id": string,
+      "name": string,
+      "homepage": string,
+      "image_id": string,
+      "order": number,
+      "desktop_link": string | null,
+      "webapp_link": string | null,
+      "app_store": string | null,
+      "play_store": string | null,
+      "mobile": {
+        deep_link: string | null,
+        universal_link: string | null
+      }
+      "injected": [
+        {
+          "namespace": string,
+          "injected_id": string
+        }
+      ] | null
+    }
```

Comment:
Do we need ALL of this information in all cases?

Can we shorten the names of the fields to save on bandwidth?

Can we sniff what platform the app is on and only request the Apple, Google, or mobile store links accordingly?

Can we request mobile links/etc. when the wallet is clicked instead of in the initial download?